firefox-ios/firefox-ios-tests/Tests/XCUITests/MicrosurveyTests.swift

```diff
@@ -84,9 +84,11 @@ final class MicrosurveyTests: BaseTestCase {
         app.buttons[AccessibilityIdentifiers.FirefoxHomepage.OtherButtons.privateModeToggleButton]
         let homepageToggleButtonIpad = app.buttons[AccessibilityIdentifiers.Browser.TopTabs.privateModeButton]
         if !iPad() {
+            mozWaitForElementToExist(homepageToggleButtonIphone)
```

Comment:
Do we have to add an if condition or this is going to work OS version backwards too?

Reply:
This change works on iOS 17, 16 and 15 as well, so there's no need for an `if` statement on the iOS version.